KUBERNETES.md

```diff
@@ -0,0 +1,231 @@
+own local Kubernetes environment.
+
+## Setup
+
+You'll need to ensure that the following software is installed:
+
+* [Docker](https://www.docker.com/)
+* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl)
+* [Minikube](https://github.com/kubernetes/minikube) v0.25.0 or later (Verify with `minikube version`)
+* [Helm](https://github.com/kubernetes/helm)
+* [reactive-cli](https://developer.lightbend.com/docs/reactive-platform-tooling/latest/cli-installation.html#install-the-cli) 0.9.0 or later (Verify with `rp version`)
+
+## Deploy
+
+Once you've setup your environment, it's time to deploy the application. There are two ways you can do this:
+
+1) Developer Workflow, targeted at developers. This allows you to deploy all of your services with a single SBT task.
+
+2) Operations Workflow, targeted at operations. This matches more closely how you would deploy a Lagom system in production.
+
+### Developer Workflow
+
+> Note that this is an alternative to the Operations workflow documented below.
+
+The following workflow is well suited for development purposes. It will build Docker images for each subproject, generate Kubernetes
+YAML, and deploy them into your Minikube using `kubectl`. It also installs the [Reactive Sandbox](https://github.com/lightbend/reactive-sandbox)
+in your Minikube, providing the required dependencies for Kafka, Cassandra, and Elasticsearch.
+
+> Want a fresh Minikube? Run `minikube delete` before the steps below.
+
+```bash
+minikube addons enable ingress
+minikube start --memory 6000
+
+sbt "deploy minikube"
+```
+
+> Open the URL this command prints in the browser
+
+```bash
+echo "http://$(minikube ip)"
+```
+
+You should now be able to use the application with your browser. Please note that since Minikube uses a self-signed
+certificate for TLS, it will result in a browser warning.
+
+### Operations Workflow
+
+> Note that this is an alternative to the Development workflow documented above.
+
+This workflow matches more closely how an application would be deployed in a production environment. In this method,
+the images are built by SBT. The [reactive-cli](https://github.com/lightbend/reactive-cli) is then
+used to generate YAML for deployment to the Kubernetes cluster. The steps below focus on Minikube, but could be used on
+a real Kubernetes cluster given access to a Docker registry and configured `kubectl`.
+
+> Want a fresh Minikube? Run `minikube delete` before the steps below.
+
+##### Start Minikube
+
+```bash
+minikube addons enable ingress
+minikube start --memory 6000
+```
+
+##### Setup Docker engine context to point to Minikube
+
+```bash
+eval $(minikube docker-env)
+```
+
+##### Install Reactive Sandbox (optional; skip if you have your own Cassandra, Kafka, and Elasticsearch clusters) 
+
+The `reactive-sandbox` includes development-grade (i.e. it will lose your data) installations of Cassandra, 
```

Comment:
Maybe, "The `reactive-sandbox` allows you to easily test your deployment with Cassandra, Kafka, Zookeeper, and Elasticsearch without using production versions.  Because it is a sandbox, it does not persist data across restarts. The sandbox is packaged as a Helm chart for easy installation into your Kubernetes cluster."

Reply:
incorporated